acurl_ng/README.md

```diff
@@ -0,0 +1,34 @@
+# Acurl_NG (Next Generation)
+
+Acurl_NG is a Cython implementation of the pre-existing acurl implementation, previously built entirely in C.
```

Comment:
```suggestion
Acurl_NG is a Cython refactoring/rewrite of an earlier iteration of the asynchronous curl wrapper concept, which was written in C for the mite project by Tony Simpson <agjasimpson@gmail.com>.
```

I think this is the right way to acknowledge the debt that this code owes to Tony's design without misattributing any of the current interation's imperfections to him by naming him as author.

Reply:
Reworded this as suggested